runtime/src/lib.rs

```diff
@@ -131,7 +131,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // The version of the runtime specification. A full node will not attempt to use its native
     //   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
-    spec_version: 2_000,
+    spec_version: 2_100,
```

Comment:
We should change the runtime `Cargo.toml` too

Reply:
Done, thanks!